sdk/keyvault/internal/challenge_policy.go

```diff
@@ -161,7 +123,7 @@ func (c *challengePolicyError) Unwrap() error {
 var _ errorinfo.NonRetriable = (*challengePolicyError)(nil)
 
 // sets the k.scope and k.tenantID from the WWW-Authenticate header
```

Comment:
Looks like this comment needs updating.

Also, maybe it's "updateScopeAndTeneant", not "find" (which implies that it doesn't change anything).